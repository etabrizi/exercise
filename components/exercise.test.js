import * as React from 'react';
import Exercise from './exercise';
import renderer from 'react-test-renderer';

const Props = {
    transcript: '<p>Here is some html<p>',
    infoToggle: () => { },
    title: 'Pull up'
}

describe('Exercise component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Exercise {...Props} />)
            .toJSON();
        expect(tree).toMatchSnapshot()
    });
});







