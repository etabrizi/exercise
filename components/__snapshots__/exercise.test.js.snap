// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exercise component renders correctly 1`] = `
<div
  className="jsx-2431219986 open"
>
  <h3
    className="jsx-2431219986"
  >
    Pull up
  </h3>
  <p>
    Here is some html
  </p>
  <p />
</div>
`;
